jwst/cube_build/ifu_cube.py

```diff
@@ -192,6 +192,14 @@ def define_cubename(self):
                         '_single_s3d.fits'
 # ________________________________________________________________________________
             elif self.instrument == 'NIRSPEC':
+
```

Comment:
This block of code is the only real change to the `cube_build` step. All the other edits are fixes to the cosmetics of some log messages.